src/Basho/Riak/Bucket.php

```diff
@@ -63,7 +67,9 @@ public function getName()
      * Returns the buckets R-value If it is set,
      * otherwise return the R-value for the client.
      *
-     * @return integer
+     * @param null $r
+     *
+     * @return int
      */
     public function getR($r = null)
```

Comment:
To be honest, I wish I knew. I can't think of a situation where things would blow up if I removed the param and the code that just returns the param, you?